### DISCUSSION TITLE
Welcome to coord-sim Discussions!

### DISCUSSION THREAD

**Discussion by stefanbschneider** [RealVNF/coord-sim#207] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AlexPasqua** [comment]
> To get started, comment below with an introduction of yourself and tell us about what you do with this community.

Hello, I'm a MSc student in Computer Science and I'm very interested in Artificial Intelligence!
I'm currently working on my thesis about reinforcement learning in the context of virtual network functions placement. I'm trying to explore this tool to see if it's suitable for my work 😄

**Reply by stefanbschneider** [reply]
That's great to hear! We have similar interests/backgrounds then :)
